src/test/java/io/jenkins/plugins/util/AgentFileVisitorTest.java

```diff
@@ -131,16 +131,16 @@ private FileSystemFacade createFileSystemFacade(final boolean followLinks, final
 
     @Override
     protected StringScanner createSerializable() {
-        return new StringScanner(PATTERN, "UTF-8", true, createFileSystemFacade(true));
+        return new StringScanner(PATTERN, "UTF-8", true,true, createFileSystemFacade(true));
     }
 
     static class StringScanner extends AgentFileVisitor<String> {
         private static final long serialVersionUID = -6902473746775046311L;
         private int counter = 1;
 
         @VisibleForTesting
-        protected StringScanner(final String filePattern, final String encoding, final boolean followSymbolicLinks, final FileSystemFacade fileSystemFacade) {
-            super(filePattern, encoding, followSymbolicLinks, fileSystemFacade);
+        protected StringScanner(final String filePattern, final String encoding, final boolean followSymbolicLinks,boolean errorOnEmptyFiles, final FileSystemFacade fileSystemFacade) {
```

Comment:
A new test with `error=false` is required as well

Reply:
Resolved and thanks for the comment